products/rhel8/profiles/stig.profile

```diff
@@ -822,6 +822,7 @@ selections:
     # RHEL-08-030640
 
     # RHEL-08-030650
+    - aide_check_audit_tools
```

Comment:
Add this rule to the stability test profile:
https://github.com/ComplianceAsCode/content/blob/master/tests/data/profile_stability/rhel8/stig.profile
https://github.com/ComplianceAsCode/content/blob/master/tests/data/profile_stability/rhel8/stig_gui.profile